clang.md

```diff
@@ -1,54 +1,73 @@
 # Compiling C/C++ to WebAssembly
 
-## Rolling your own compiler
+Many high level languages already support compilation to WebAssembly
+through the experimental LLVM backend. Unfortunately, it is a tedious
```

Comment:
> experimental LLVM backend

Could you please add a link to further documentation on this? Why is it "experimental"?